sources/packages/backend/apps/api/src/route-controllers/student/models/student.dto.ts

```diff
@@ -214,6 +214,7 @@ export class ApplicationSummaryAPIOutDTO {
   submitted?: Date;
   status: ApplicationStatus;
   parentApplicationId: number;
+  submittedDate?: Date;
```

Comment:
Noticed that there is `submitted` property here which is mapped to assessment submitted date and seems obsolete.

![image](https://github.com/user-attachments/assets/7f971416-f225-4ecf-a20e-2effcd0dc332)

![image](https://github.com/user-attachments/assets/323417ed-7e44-48b0-85c0-311cb999603c)

Reply:
I will keep only one.

Reply:
Removed the old one.